src/app/services/chat.service.ts

```diff
@@ -183,6 +183,13 @@ export class ChatService {
       };
     });
   }
+
+  // groups 
+
+  getAllGroups(id:any){
```

Comment:
for admin we need all groups , but for user we need only user groups, for admin call "/groups" only dont pass id

Reply:
done